task2/main.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	"fmt"
+	"sort"
+)
+
+func MiniMaxSum(arr []int) (int, int) {
+	var _arr []int
```

Comment:
why do you use underscore when you init variable?

Reply:
We use _arr, because its a new arr of sum. Then we sort sum from min to max. Output: _arr[0], _arr[arrayLength-1]